hack/lib/golang.sh

```diff
@@ -785,6 +788,18 @@ kube::golang::build_binaries() {
     goasmflags="-trimpath=${KUBE_ROOT}"
     gogcflags="${GOGCFLAGS:-} -trimpath=${KUBE_ROOT}"
 
+    # extract tags if any specified in GOFLAGS
+    # shellcheck disable=SC2001
+    gotags=$(echo "${GOFLAGS:-}" | sed -e 's|.*-tags=\([^-]*\).*|\1|')
+    # strip double quotes
+    gotags=${gotags//\"}
+    # strip single quotes
+    gotags=${gotags//\'}
+    # trim leading and trailing whitespaces
+    gotags=$(echo "$gotags" | awk '{ gsub(/^[ \t]+|[ \t]+$/, ""); print }')
```

Comment:
leading and trailing whitespace is illegal, as are the quotes, we can drop this and just `,` join with `selinux`
requires go 1.13+ but we're on that